### PR TITLE
About freefeed

### PR DESCRIPTION
**Pull request by dsumin**
Update /about page
Create /dev page
Add references to the /about page to /welcome

### CODE REVIEW

File: src/components/about.jsx [hunk 1]
```diff
@@ -1,14 +1,56 @@
...
+            <p>FreeFeed v.1 is scheduled for release in spring 2016. This version will include search and hashtag support.</p>
+
+            <p>You can <b><Link to='/signup'>sign up</Link></b> now and use the beta version today.</p>
+
+            <h3>Help us build better FreeFeed</h3>
+            <p>We are <Link to='/dev'>looking for volunteers</Link> to help us with the development of FreeFeed.net.</p>
+
+            <h3>Important pages</h3>
+
+            <p><a href="https://beta.freefeed.net/support" target="_blank">https://beta.freefeed.net/support</a> -
```
Comment (indeyets) [inline src/components/about.jsx]: `<Link to="/support">` maybe?

File: src/components/footer.jsx [hunk 2]
```diff
@@ -3,6 +3,6 @@ import React from 'react'
 export default (props) => (
   <footer className='footer p-footer'>
     &copy; FreeFeed 0.6.3 (February 8, 2016)<br/>
-    <a href="https://about.freefeed.net" target="_blank">About</a> | <a href="http://news.freefeed.net" target="_blank">Public News</a> | <a href="https://twitter.com/freefeednet" target="_blank">Twitter</a> | <a href="http://freefeed.reformal.ru/" target="_blank">Reformal: suggest new features</a>
+    <a href="https://beta.freefeed.net/about" target="_blank">About</a> | <a href="http://beta.freefeed.net/freefeed" target="_blank">News</a> | <a href="https://twitter.com/freefeednet" target="_blank">Twitter</a> | <a href="http://freefeed.reformal.ru/" target="_blank">Reformal: suggest new features</a>
```
Comment (clbn) [inline src/components/footer.jsx]: It has to be `/about` without a domain, we don't want to change later. And `<Link>`, not `<a>`.

File: src/components/welcome.jsx [hunk 3]
```diff
@@ -3,9 +3,13 @@ import {Link} from 'react-router'
 
 export default _ => (
   <div className='box-body'>
-    <h3>Welcome to FreeFeed</h3>
-    <p>&lt;Here's the description of what FreeFeed is. We need a copywriter!&gt;</p>
-    <p>If you already have an account, you might want to <Link to='/signin'>sign in</Link>.</p>
-    <p>Otherwise, consider <Link to='/signup'>signing up</Link> to enjoy all the fun stuff.</p>
-  </div>
+    <h3>Welcome to FreeFeed.net</h3>
```
Comment (clbn) [inline src/components/welcome.jsx]: Indentation (we have 2 spaces both for HTML and JS, not 4).

File: src/components/welcome.jsx [hunk 4]
```diff
@@ -3,9 +3,13 @@ import {Link} from 'react-router'
...
+      <p>FreeFeed is an <a href="https://github.com/FreeFeed/" target="_blank">open-source</a> social
```
Comment (clbn) [inline src/components/welcome.jsx]: We don't need trailing slash in the URL. And I'd argue we need that link here at all. The two important links in this text must be "Sign up" and "Sign in" (and maybe "About", but only barely), so we don't want to make them more competition with a GitHub link.

File: src/components/dev.jsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import {Link} from 'react-router'
+
+export default (props) => (
+  <div className='box'>
+    <div className='box-header-timeline'>
+      Help us build FreeFeed
+    </div>
+    <div className='box-body'>
+        <div>
```
Comment (clbn) [inline src/components/dev.jsx]: Indentation.

File: src/index.jsx [hunk 6]
```diff
@@ -49,6 +50,7 @@ ReactDOM.render(
       <Route path='/' component={Layout}>
         <IndexRoute name='home' component={Home} onEnter={boundRouteActions('home')}/>
         <Route path='about' component={About} />
+        <Route path='dev' component={Dev} />
```
Comment (clbn) [inline src/index.jsx]: If we want to use `/dev`, we should add it to the username blacklist in `freefeed-server`.

**Comment by clbn** [comment]
I left some comments in-code. I also have some thoughts about the texts, but we can discuss them separately.

File: src/components/about.jsx [hunk 7]
```diff
@@ -1,14 +1,56 @@
...
   <div className='box'>
     <div className='box-header-timeline'>
-      Other page
+      About FreeFeed
     </div>
     <div className='box-body'>
-      <div>
-        Here be more dragons
-      </div>
+    <div>
```
Comment (clbn) [inline src/components/about.jsx]: Indentation (unlike other cases, this one should be moved to the right).

File: src/components/footer.jsx [hunk 8]
```diff
@@ -1,8 +1,9 @@
 import React from 'react'
+import {Link} from 'react-router'
 
 export default (props) => (
   <footer className='footer p-footer'>
     &copy; FreeFeed 0.6.3 (February 8, 2016)<br/>
-    <a href="https://about.freefeed.net" target="_blank">About</a> | <a href="http://news.freefeed.net" target="_blank">Public News</a> | <a href="https://twitter.com/freefeednet" target="_blank">Twitter</a> | <a href="http://freefeed.reformal.ru/" target="_blank">Reformal: suggest new features</a>
+    <Link to='/about' target="_blank">About</Link> | <Link to='/freefeed'>News</Link> | <a href="https://twitter.com/freefeednet" target="_blank">Twitter</a> | <a href="http://freefeed.reformal.ru/" target="_blank">Reformal: suggest new features</a>
```
Comment (clbn) [inline src/components/footer.jsx]: We don't need `target` attribute on the internal links.

Comment (clbn) [inline src/components/footer.jsx]: And, by the way, we should use `"` for HTML/JSX (`'` is for JS).

File: src/components/dev.jsx [hunk 9]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import {Link} from 'react-router'
+
+export default (props) => (
+  <div className='box'>
+    <div className='box-header-timeline'>
+      Help us build FreeFeed
+    </div>
+    <div className='box-body'>
+    <div>
```
Comment (clbn) [inline src/components/dev.jsx]: Indentation again (now it's too far to the left).

File: src/components/about.jsx [hunk 10]
```diff
@@ -1,13 +1,55 @@
...
+        <p><img src="/assets/images/sample.png"></img></p>
+
+
+        <h3>Why FreeFeed.net?</h3>
+
+        <p>FreeFeed is being built as a replacement for <a href="https://en.wikipedia.org/wiki/FriendFeed" target="_blank">FriendFeed</a> —
+        the real-time aggregator and social network where 'likes' for user generated content were
+        implemented for the first time.</p>
+
+        <p>After Facebook had acquired FriendFeed and announced its plan to shutdown down the website on April
```
Comment (f355) [inline src/components/about.jsx]: "shutdown down".
Also, "announced the plan" sounds better than "announced its plan", I reckon.

Comment (clbn) [inline src/components/about.jsx]: +1

**Pull request merged**